app/domain/pusher/pusher.go

```diff
@@ -249,7 +249,7 @@ func (h *MetricsPusher) sendBatch(batch []*types.ResourceTags) error {
 	}
 
 	ts := h.formatMetrics(batch)
-	log.Ctx(h.ctx).Info().
+	log.Ctx(h.ctx).Debug().
```

Comment:
I had changed this to Info because otherwise, it's difficult for users to know if the webhook-servers are working. I think ideally we'd be checking some /metrics endpoint and/or getting info in the cluster details page, but I think users tend to check logs first.

this also doesn't log very much; would probably only log once a minute a most, and only when data is added. but, I'm good with changing to debug if we think we have better positive confirmation elsewhere